### PR TITLE
Improved warnings for lua-package

### PR DESCRIPTION
**Pull request by CrosRoad95**
Make warnings user friendly

### Before
image

### After
image
image
image
image

### CODE REVIEW

**Review by qaisjp — CHANGES_REQUESTED** [review]
This misnumbers errors for empty files or files that return nil. See example resource [nil_require.zip](https://github.com/multitheftauto/mtasa-blue/files/2544510/nil_require.zip).

File: Shared/mods/deathmatch/logic/lua/CLuaMain.Shared.cpp [hunk 1]
```diff
@@ -137,15 +137,18 @@ bool CLuaMain::LoadLuaLib(lua_State* L, SString strName, SString& strError)
 
         if (lua_type(L, -1) == LUA_TNIL)
         {
-            strError += "error loading module " + strName + " from file " + strFilePath + ":\n\t Module didn't return a value";
+            strError += "#3: " + ConformPath(strFilePath, "resources/") + "\n\t";
```
Comment (qaisjp) [inline Shared/mods/deathmatch/logic/lua/CLuaMain.Shared.cpp]: "Module didn't return a value" information is lost

**Review by qaisjp — CHANGES_REQUESTED** [review]

File: Shared/mods/deathmatch/logic/lua/CLuaMain.Shared.cpp [hunk 2]
```diff
@@ -99,14 +99,14 @@ bool CLuaMain::LoadLuaLib(lua_State* L, SString strName, SString& strError, bool
     ReplaceOccurrencesInString(strPath, "/", "");
     ReplaceOccurrencesInString(strPath, ".", "/");
 
-    #ifdef MTA_CLIENT
-        SString strResPath = m_pResource->GetResourceDirectoryPath(ACCESS_PUBLIC, "");
-    #else
-        SString strResPath = m_pResource->IsResourceZip() ? m_pResource->GetResourceCacheDirectoryPath() : m_pResource->GetResourceDirectoryPath();
-    #endif
+#ifdef MTA_CLIENT
```
Comment (qaisjp) [inline Shared/mods/deathmatch/logic/lua/CLuaMain.Shared.cpp]: ifdef changes here need to be reverted.  it's annoying that clang-format does this

**Review by patrikjuvonen — APPROVED** [review]
I believe this PR is ready to be merged with `feature/lua-packages`.

**Pull request merged**